### PR TITLE
Stop removing the MVAR table

### LINKED ISSUE #580: Don't kill MVAR table in fix-font
We've re-thought the fontbakery check that complained about the MVAR table (https://github.com/googlefonts/fontbakery/issues/3400) but we still drop the MVAR table in fix-font. The problem with this is that fonts instantiated from a VF with no MVAR table have the wrong vertical metrics.

### PR DESCRIPTION
**Pull request by madig**
Closes #580.

### CODE REVIEW

File: Lib/gftools/fix.py [hunk 1]
```diff
@@ -613,7 +612,6 @@ def fix_font(font, include_source_fixes=False, new_family_name=None, fvar_instan
         fix_unhinted_font(font)
 
     if "fvar" in font:
-        remove_tables(font, ["MVAR"])
```
Comment (m4rc1e) [inline Lib/gftools/fix.py:616]: By dropping this, we won't remove the other redundant tables. Would prefer

```PYTHON
remove_tables(font)
```

Comment (m4rc1e) [reply Lib/gftools/fix.py:616]: Sorry, you are correct. This func already gets called earlier.

**Review by m4rc1e — APPROVED** [review]

**Comment by m4rc1e** [comment]
I'll chat to Chris before I merge this. I may need to update fontbakery as well.

**Comment by chrissimpkins** [comment]
Marc and I discussed this today.  OK to merge.  Note that we will begin to add MVAR tables to new VF families and existing VF families that update their gftools dependency version as of the release that includes these changes.

fontbakery was changed to stop failing VF files that contain MVAR in https://github.com/googlefonts/fontbakery/commit/115b28575f9bf48f45a089efebdd018da5be3123.  As I understand it, there is no longer a check for MVAR at all in fontbakery.  There will be no WARN or FAIL for VFs with MVAR.

**Pull request merged**